File: Data Structures & Algorithms/is-palindrome/submission-2.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        right = len(s) - 1
        left = 0
        while left <= right:
            if  not s[left].isalnum():
                left += 1
            if not s[right].isalnum():
                right -= 1
            print(f"{s[left]}, {s[right]}")
            if s[left].lower() != s[right].lower():
                return False
            right -= 1
            left += 1
        return True

File: Data Structures & Algorithms/two-integer-sum-ii/submission-2.js
class Solution {
    /**
     * @param {number[]} numbers
     * @param {number} target
     * @return {number[]}
     */
    twoSum(numbers, target) {
        let left = 0;
        let right = numbers.length - 1;
        while (left < right){
            let sum = numbers[left] + numbers[right]
            if (sum === target){
                return [numbers[left], numbers[right]];
            }
            else if (sum < target){
                left++;
            
            }
            else {
                right--;
            }
        }
    }
}
